integration-tests/tests/src/utils/import-app.ts

```diff
@@ -92,7 +92,8 @@ export async function importApp(
 ): Promise<{ appId: string; baseUrl: string; databaseName: string }> {
   const { baseUrl, appImporterUrl } = getUrls();
 
```

Comment:
we could add an optional `databaseName` to `importApp` and `importAppBefore` but I'm not sure if that's the cleanest solution..